src/components/article.jsx

```diff
@@ -31,6 +34,39 @@ const Article = () => {
         })
     }, [article_id, setComLoading]);
 
+    const handleUpvoteClick = () => {
+        setArticle({
+          ...article,
+          votes: article.votes + 1,
```

Comment:
we don't necessarily need to be using the whole article here, we could just have a votes state where wouldn't need to be spreading the whole article